### ISSUE TITLE
Did you found any bugs?

### ISSUE THREAD

**Issue by sebastian-philipp** [fkpwolf/kubic-kvm#1]
Hey, did you found any issues that would be worth fixing in the official clone [kubic-project/kubic-terraform-kvm]https://github.com/kubic-project/kubic-terraform-kvm)?

**Comment by fkpwolf** [comment]
No, just found the repository. But this repo is an example for pci-passthough feature. You can check https://github.com/fkpwolf/terraform-provider-libvirt/tree/pci-passthough.
